client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx

```diff
@@ -147,6 +148,13 @@ export const ApplicationsTable: React.FC = () => {
     refetch: fetchApplications,
   } = useFetchApplications();
 
+  const {
+    archetypes,
+    isFetching: isFetchingArchetypes,
+    error: archetypesFetchError,
+    refetch: fetchArchetypes,
+  } = useFetchArchetypes();
```

Comment:
Just a thought -- when the application assessment + analysis tables are combine, it would be good to rework the data fetching & data cross-references into its own hook.  That way the hook can do something like...

```js
fancyApplication = {
  ...applicaiton,
  archetypes: mapRefsToObject(applicaiton.archetypes, archetypes),
  isReviewed: ....,
  hasReviewedArchetype: ...,
}
```

...then the component code can just wire things together

Reply:
+1